src/main/java/uk/gov/register/resources/RedirectResource.java

```diff
@@ -59,12 +108,17 @@ public Response getRecordEntriesRedirect(
         return redirectByPath(request, "/record/", "/records/");
     }
 
-
-
     private static ResponseBuilder movedPermanently(URI location) {
         return status(Response.Status.MOVED_PERMANENTLY).location(location);
     }
 
+    public static Response temporaryRedirectByPath(@Context HttpServletRequest request, String oldPath, String newPath) {
```

Comment:
we could use overloading to DRY this up a bit. i.e. add a version of `redirectByPath` that takes a `responseStatus` argument and default the current one to `MOVED_PERMANENTLY`.